llvm/lib/Target/AArch64/AArch64TargetTransformInfo.cpp

```diff
@@ -914,6 +914,23 @@ AArch64TTIImpl::getIntrinsicInstrCost(const IntrinsicCostAttributes &ICA,
     }
     break;
   }
+  case Intrinsic::experimental_vector_match: {
```

Comment:
Could you pull the cost model changes out into a separate PR please? I think they make sense as a standalone change. Also, it would be good to add some tests in llvm/test/Analysis/CostModel/AArch64/sve-intrinsics.ll. You can look at `@reductions` as an example.

Reply:
Thanks, I've done so in #118512. Please let me know if you have any other comments!